js/game.js

```diff
@@ -204,12 +222,10 @@ function finishSubgame() {
 function rivalBet() {
     let action = game.rival.bet(game);
     switch (action) {
-        case Bets.PASS: // note: only if user passed, otherwise it's folding!
-            game.lastAction = LastAction.RIVAL_PASS;
-            game.betsAreSet = true;
-            break;
-        case Bets.FOLD:
-            game.lastAction = LastAction.RIVAL_FOLD;
+        case Bets.FOLD, Bets.PASS:
+            // AI does not currently return PASS, it's collapsed with FOLD
+            // so we need to tell it apart from context
```

Comment:
I'm not sure that this is an actual case, because the ai currently has this check before folding:

```js
if (game.playerBet <= 1) {
    return Bets.CHECK;
}
```
so it should go though the `CHECK` case that it's more or less the same as `PASS`.

Also `case Bets.FOLD, Bets.PASS` it's equivalent to `case Bets.PASS` due to the usage of the [comma operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator), so that feature wasn't being used.

I'll add a commit with your intended fix and, later, we can check if this is doing anything or not.